db/migrate/20250428154055_create_sign_in_certificates.rb

```diff
@@ -0,0 +1,14 @@
+class CreateSignInCertificates < ActiveRecord::Migration[7.2]
+  def change
+    create_table :sign_in_certificates, id: :uuid do |t|
+      t.references :config, polymorphic: true, index: true, null: false
```

Comment:
Should the configs store the cert reference? Or do you think it's better for the cert objects to store the config reference?